trl/trainer/sft_trainer.py

```diff
@@ -721,6 +712,9 @@ def __init__(
         use_flash_attention = model.config._attn_implementation in [
```

Comment:
nit: should this be promoted to a constant like `FLASH_ATTENTION_VARIANTS` so we can then populate the logger warnings with a single variable instead off having to manually edit them each time?